advanced/wallets/react-wallet-v2/src/data/Bip122Data.ts

```diff
@@ -0,0 +1,43 @@
+/**
+ * Chains
+ */
+export const BITCOIN_MAINNET = {
+  'bip122:000000000933ea01ad0ee984209779ba4ff763ae46a2a6c172b3f1b60a8ce26f': {
```

Comment:
Should be `bip122:000000000019d6689c085ae165831e93` based on CAIP-2 spec? https://github.com/ChainAgnostic/namespaces/blob/main/bip122/caip2.md#test-cases